cmd/kubeadm/app/constants/constants.go

```diff
@@ -121,6 +121,9 @@ var (
 
 	// MinimumControlPlaneVersion specifies the minimum control plane version kubeadm can deploy
 	MinimumControlPlaneVersion = version.MustParseSemantic("v1.7.0")
+
+	// MinimumCSRAutoApprovalClusterRolesVersion
+	MinimumCSRAutoApprovalClusterRolesVersion = version.MustParseSemantic("v1.7.0-alpha.3")
```

Comment:
Thanks for that catch! Should have been v1.8.0-alpha.3 indeed as the PR just merged